pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerStarter.java

```diff
@@ -81,8 +72,6 @@
 
   public ControllerStarter(ControllerConf conf) {
     config = conf;
-    component = new Component();
-    controllerRestApp = new ControllerRestApplication(config.getQueryConsole());
```

Comment:
(optional) The executorservice  threads are named as 'restlet....'. We can change that to avoid possible confusion